data/tlg1799/tlg016/tlg1799.tlg016.1st1K-grc1.xml

```diff
@@ -0,0 +1,1336 @@
+                    <persName>Bruce Robertson</persName>
+                    <resp>Technical Advisor</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Harvard College Library</authority>
+                <idno type="filename">tlg1799.tlg.1st1K-grc1.xml</idno>
+                <availability>
+                    <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International
+                        License</p>
+                </availability>
+                <date>2017</date>
+                <publisher>Harvard College Library</publisher>
+                <pubPlace>United States</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title xml:lang="lat">Euclidis Opera Omnia</title>
+                            <editor>
+                                <persName>
+                                    <name xml:lang="lat">Henricus Menge</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:cts:greekLit:tlg1799">Euclid</author>
+                            <imprint>
+                                <publisher>Leipzig</publisher>
+                                <pubPlace>Teubner</pubPlace>
+                                <date>1916</date>
+                            </imprint>
+                            <biblScope unit="volume">8</biblScope>
+                        </monogr>
+                        <ref
+                            target="https://archive.org/stream/euclidisoperaomn08eucluoft#page/n3/mode/2up"
+                            >Internet Archive</ref>
+                    </biblStruct>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>Text encoded in accordance with the latest EpiDoc standards</p>
+            <refsDecl n="CTS">
+                <cRefPattern matchPattern="(.+).(.+)" n="section"
+                    replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:div[@n='$2'])"/>
+                <cRefPattern matchPattern="(.+)" n="chapter"
+                    replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"
+                />
+            </refsDecl>
+        </encodingDesc>
+        <profileDesc>
+            <langUsage>
+                <language ident="grc">Greek</language>
+                <language ident="lat">Latin</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change/>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="grc" n="urn:cts:greekLit:tlg1799.tlg016.1st1K-grc1">
+                <pb n="227"/>
+                <div type="textpart" subtype="chapter" n="1">
```

Comment:
There are several chapters here that have no sections under them. I think you should add a single "section"-level div under each of these chapters to keep the citation scheme consistent.